cmd/minikube/cmd/start.go

```diff
@@ -1585,7 +1591,13 @@ func createNode(cc config.ClusterConfig, existing *config.ClusterConfig) (config
 	// Create the initial node, which will necessarily be a control plane
 	if existing != nil {
 		cp, err := config.PrimaryControlPlane(existing)
-		cp.KubernetesVersion = getKubernetesVersion(&cc)
+
+		newVer, err2 := getKubernetesVersion(&cc)
+		cp.KubernetesVersion = newVer
+
+		if err2 != nil {
+			klog.Warningf("get kubernetesVersion failed : %v", err2)
+		}
 		cp.ContainerRuntime = getContainerRuntime(&cc)
 		if err != nil {
```

Comment:
This was setup wrong from the start, but there shouldn't be any lines of code between `cp, err := config.PrimaryControlPlane(existing)` and `if err != nil {`

So this should be reordered to:
```
cp, err := config.PrimaryControlPlane(existing)
if err != nil {
        return cc, config.Node{}, err
}
newVer, err := getKubernetesVersion(&cc)
...
```
And then you can change `err2` back to `err`